tracer/src/Datadog.Trace/Logging/Internal/DatadogLoggingFactory.cs

```diff
@@ -119,6 +135,12 @@ static bool Contains(string?[]? array, string toMatch)
                                shared: true));
         }
 
+        if (config.Console is { } consoleConfig)
+        {
+            loggerConfiguration
+               .WriteTo.Sink(new ConsoleSink(consoleConfig.MessageTemplate, consoleConfig.BufferSize));
```

Comment:
We need to include the fix from https://github.com/DataDog/dd-trace-dotnet/pull/7049 too:

```suggestion
               .WriteTo.Logger(
                   lc => lc
                         .Enrich.With(new RemovePropertyEnricher(LogEventLevel.Error, DatadogSerilogLogger.SkipTelemetryProperty))
                         .WriteTo.Sink(new ConsoleSink(consoleConfig.MessageTemplate, consoleConfig.BufferSize));
```